BuildingSync/schema
# build: add gh workflow for building releases

#### Any background context you want to provide?

#### What does this PR do?
Add a github workflow that's triggered on tag pushes, which builds the webpage along with the data dict stuff and adds it to the release.
The docs html is now being built with a forked version of [this repo](https://github.com/bitfehler/xs3p). It is now much smaller in size and faster.

Also deletes the json schema b/c we aren't using it. Also deletes enumerations.json and DataDictionary.xlsx b/c these are "built" files which are included in the release.

#### How should this be manually tested?
Once merged, push a test tag (non pre-release) and verify:
1. it created the release
2. it included index.html and json and xlsx files
3. the description of the release is the most recent section of the changelog

Then try it again with a pre-release tag (ie include vx.x.x-prx) and verify that it labels the release as a pre-release

#### What are the relevant tickets?
#305 

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2021-04-15 at 8 28 46 AM" src="https://user-images.githubusercontent.com/18518728/114886277-a5752200-9dc4-11eb-867e-71c6c489958c.png">
<img width="1440" alt="Screen Shot 2021-04-15 at 8 28 33 AM" src="https://user-images.githubusercontent.com/18518728/114886283-a6a64f00-9dc4-11eb-944d-28d0ae830692.png">
